app/src/main/java/org/mozilla/fenix/tabstray/TrayPagerAdapter.kt

```diff
@@ -53,7 +55,7 @@ class TrayPagerAdapter(
                     itemView,
                     store,
                     interactor,
-                    browserStore.state.normalTabs.indexOf(selectedTab)
+                    browserStore.state.normalTabs.filter { it.isNormalTabActive(maxActiveTime) }.indexOf(selectedTab)
```

Comment:
Since we have a single RecyclerView with multiple adapters, our "index" calculation is going to be a bit complicated.

From our last meeting, we need to scroll to the selected tab even if it's in the inactive tab section as well.

So we would need to do something like:
1. Iterate through all adapters
2. Find the adapter with a tab of the matching ID and viewholder type.
3. Convert the relative adapter positive to the final adapter position.
4. Tell the RV to scroll to this new position (because we need to say `RecyclerView.scrollToPosition` and there is no `RecyclerView.Adapter.scrollToPosition`).

This will also aid in solving a scrolling regression when we land other adapters like in 21177.

Reply:
Spoke offline with Noah, we'll land this change for now and fix the scrolling problem when we land 21177 as well.